src/main/resources/application.yaml

```diff
@@ -58,3 +58,8 @@ ftp:
   downtime:
     from: ${FTP_DOWNTIME_FROM:17:00}
     to: ${FTP_DOWNTIME_TO:18:00}
+
+tasks:
+  uploadInterval: ${UPLOAD_LETTERS_INTERVAL:30000}
+  reports-cron: ${FTP_REPORTS_CRON:0 30 0 * * *}
```

Comment:
i guess question about camel case and dashes should've been in main app config. putting a reminder here